Gruntfile.js

```diff
@@ -342,7 +387,7 @@ module.exports = function(grunt) {
               `npm dedupe`,
               `npm pack`,
               `ls -l medic-${module}-0.1.0.tgz`,
-              `mv medic-*.tgz ../build/ddocs/medic/_attachments/`,
+              `mv medic-*.tgz ../build/staging/_attachments/`,
```

Comment:
These attachments are going away, right? Is that a future PR?

Reply:
Yes. Right now, there is no way to get API and Sentinel to run e2e tests. So I'm keeping the "horti" + attachments system still in place. These attachments go away when we publish in a docker registry.